ProjectSourceCode/server/index.js

```diff
@@ -451,6 +475,39 @@ app.post('/api/activities', auth, async (req, res) => {
     }
 });
 
+// Journal entry api
+app.post('/api/journal', auth, async (req, res) => {
+    try {
+      const userId = req.session.user.user_id;
+      const { 'journal-entry': journalEntry } = req.body;
+  
+      // Validate that a journal entry was provided
+      if (!journalEntry) {
+        console.error('No journal entry provided');
+        return res.status(400).redirect('/activity');
```

Comment:
Just realized this one never got changed from /activity to /journal.
```suggestion
        return res.status(400).redirect('/journal');
```